docs/modules/development/pages/rest/snmp_configuration.adoc

```diff
@@ -156,8 +156,8 @@ It also shows the version limitations, default values, and the restrictions - if
 [options="header, %autowidth"]
```

Comment:
I am unable to quote lines 153 and 154. Suggest the following:


The following table shows all supported attributes and the mapping between `snmp-info.xsd` and `snmp-config.xsd`.
It also shows the version limitations, default values, and the restrictions, if any.